src/explore-education-statistics-admin/src/pages/publication/PublicationMethodologiesPage.tsx

```diff
@@ -99,7 +99,7 @@ const PublicationMethodologiesPage = () => {
             to an external file that contains methodology details.
           </p>
 
-          {permissions.canCreateMethodologies && (
+          {permissions?.canCreateMethodologies && (
```

Comment:
Again, these shouldn't optional - we should be working with `PublicationWithPermissions` from `PublicationContext`